clusters/app.ci/release-controller/deploy-ocp-arm64-controller.yaml

```diff
@@ -0,0 +1,299 @@
+##################################################################################
+#                                DO NOT EDIT
+# File generated during execution of: generate-release-controllers.py
+##################################################################################
+
+
+#---------------------------------------------------------------------------------
+# osd_rc_deployments.py added the following resource
+#---------------------------------------------------------------------------------
+
+# Resources required to deploy release controller instances on
+# the app.ci clusters.
+# Bootstrap the environment for the amd64 tests image.  The caches require an amd64 "tests" image to execute on
```

Comment:
`s/amd64/arm64/g` ?

Reply:
We actually need the `amd64` test image because the build clusters are what actually use it and they are all `amd64` based.